hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieIndexConfig.java

```diff
@@ -244,6 +244,12 @@ public class HoodieIndexConfig extends HoodieConfig {
       .defaultValue("true")
       .withDocumentation("Similar to " + BLOOM_INDEX_UPDATE_PARTITION_PATH_ENABLE + ", but for simple index.");
 
+  public static final ConfigProperty<String> GLOBAL_INDEX_DEDUP_PARALLELISM = ConfigProperty
```

Comment:
This and other parallelism configs seem like good candidates for `HoodieInternalConfig`. This is not going to be used by the users often. Their expectation would be to dedup as fast as we can. Don't have to do it in this patch but just want to know your thoughts?

Reply:
not very clear at the moment, given this is still tunable depends on the data's update ratio. it may stay as a infrequently used one like `hoodie.markers.delete.parallelism`

Reply:
ok, let's keep it this way. we can revisit later if necessary.

Reply:
lets make sure this is tagged an adv config? or not exposed to user by default. User should n't have to tune this.

Reply:
also calling this `deduping` overloads the meaning a bit. - we are not removing the duplicates per see, right? We only ensure the tagging routes it to the right record? `"hoodie.global.index.reconcile.parallelism"`